Assets/Scripts/Database/DialogueClipDb.cs

```diff
@@ -0,0 +1,82 @@
+using System.Data;
+
+namespace Database
+{
+    
+    public class DialogueClipDb : SqliteHelper
+    
+    {
+        private const string TableName = "DialogueLines";
```

Comment:
```suggestion
        private const string TableName = "DialogueClips";
```
The table "DialogueLines" already exists

Reply:
Oh yeah couldn't be a copy&paste *hust*